Formula/texlive.rb

```diff
@@ -0,0 +1,499 @@
+    ]
+
+    args << if OS.mac?
+      "--without-x"
+    else
+      # Make sure xdvi uses xaw, even if motif is available
+      "--with-xdvi-x-toolkit=xaw"
+    end
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+    system "make", "texlinks"
+
+    # Create tlmgr config file.  This file limits the actions that the user
+    # can perform in 'system' mode, which would write to the cellar.  'tlmgr' should
+    # be used with --usermode whenever possible.
+    (share/"texmf-config/tlmgr/config").write <<~EOS
+      allowed-actions=candidates,check,dump-tlpdb,help,info,list,print-platform,print-platform-info,search,show,version,init-usertree
+    EOS
+
+    # Delete some binaries that are provided by existing formulae as newer versions.
+    rm bin/"latexindent" # provided by latexindent formula
+    rm bin/"latexdiff" # provided by latexdiff formula
+    rm bin/"latexdiff-vc" # provided by latexdiff formula
+    rm bin/"latexrevise" # provided by latexdiff formula
+
+    # Wrap some Perl scripts in an env script so that they can find dependencies
+    env_script_files = [
+      "crossrefware/bbl2bib.pl",
+      "crossrefware/bibdoiadd.pl",
+      "crossrefware/bibmradd.pl",
+      "crossrefware/biburl2doi.pl",
+      "crossrefware/bibzbladd.pl",
+      "crossrefware/ltx2crossrefxml.pl",
+      "ctan-o-mat/ctan-o-mat.pl",
+      "ctanify/ctanify",
+      "ctanupload/ctanupload.pl",
+      "exceltex/exceltex",
+      "latex-git-log/latex-git-log",
+      "pax/pdfannotextractor.pl",
+      "ptex-fontmaps/kanji-fontmap-creator.pl",
+      "purifyeps/purifyeps",
+      "svn-multi/svn-multi.pl",
+      "texdoctk/texdoctk.pl",
+      "ulqda/ulqda.pl",
+    ]
+
+    env_script_files.each do |perl_script|
+      bin_name = File.basename(perl_script, ".pl")
+      rm bin/bin_name
+      (bin/bin_name).write_env_script(share/"texmf-dist/scripts"/perl_script, PERL5LIB: ENV["PERL5LIB"])
+    end
+
+    # Wrap some Python scripts so they can find dependencies.
+    python_path = libexec/Language::Python.site_packages("python3")
+    ENV.prepend_path "PYTHONPATH", python_path
+    (bin/"pygmentex").write_env_script(share/"texmf-dist/scripts/pygmentex/pygmentex.py",
+      PYTHONPATH: ENV["PYTHONPATH"])
+    (bin/"pythontex").write_env_script(share/"texmf-dist/scripts/pythontex/pythontex.py",
+      PYTHONPATH: ENV["PYTHONPATH"])
+
+    # Rewrite shebangs in some Python scripts so they use brewed Python.
+    python_shebang_rewrites = [
+      "latex-papersize/latex-papersize.py",
+      "lilyglyphs/lily-glyph-commands.py",
+      "lilyglyphs/lily-image-commands.py",
+      "lilyglyphs/lily-rebuild-pdfs.py",
+      "texliveonfly/texliveonfly.py",
+    ]
+
+    python_shebang_rewrites.each do |python_script|
+      rewrite_shebang detected_python_shebang, share/"texmf-dist/scripts"/python_script
+    end
+
+    # Delete ebong on Linux, because it requires Python 2
+    rm bin/"ebong" if OS.linux?
```

Comment:
I don't think this needs to be conditional. I wouldn't want to have mixed python dependencies on macOS.

Reply:
That makes sense - I'll change it.